src/WebJobs.Extensions.DurableTask/EtwEventSource.cs

```diff
@@ -446,6 +447,18 @@ private EtwEventSource()
             this.WriteEvent(227, TaskHub, AppName, SlotName, FunctionName, InstanceId, OperationName, OperationId, FunctionType, ExtensionVersion, IsReplay);
         }
 
+        [Event(228, Level = EventLevel.Warning)]
+        public void TokenRenewalFailed(
```

Comment:
Please also add the `TaskHub` and `AppName` parameters. These parameters need to exist for all of these events.

Reply:
Added `SlotName` too